cit3.0-web/src/App.js

```diff
@@ -1,82 +1,136 @@
-          <Footer />
+    <AuthStateContext.Consumer>
+      {() => (
+        <div className="app-container">
+          <LoadingBar
+            style={{
+              zIndex: 9999,
+              backgroundColor: "#fcba19",
+              height: "3px",
+            }}
+          />
+          <Header header={header} />
+          <Router>
+            <Switch>
+              <Redirect exact from="/" to="/dashboard" />
+              <AppRoute
+                title={getTitle("Login")}
+                path="/login"
+                component={Login}
+              />
+              <AppRoute
+                title={getTitle("Logout")}
+                path="/logout"
+                component={Logout}
+              />
+              <AppRoute
+                title={getTitle("Access Denied - Login to continue")}
+                path="/forbidden"
+                component={AccessDenied}
+              />
+              <AppRoute
+                title={getTitle("Opportunity Dashboard")}
+                path="/dashboard"
+                component={EDODashboard}
+              />
+              <AppRoute
+                protected
+                path="/addOpportunity/"
+                title={getTitle("Add Opportunity Parcel")}
+                layout={AuthLayout}
+                component={AddOpportunity}
+              />
+              <AppRoute
+                protected
+                path="/addOpportunity/siteDetails"
+                title={getTitle("Opportunity Site Information")}
+                layout={AuthLayout}
+                component={SiteInfomation}
+              />
+              <AppRoute
+                protected
+                path="/addOpportunity/propDetails1"
+                title={getTitle("Add Property Details")}
+                layout={AuthLayout}
+                component={PropertyDetails1}
+              />
+              <Route
+                protected
+                path="/addOpportunity/propDetails2"
+                title={getTitle("Add Additional Details")}
+                layout={AuthLayout}
+                component={PropertyDetails2}
+              />
+              <AppRoute
+                protected
+                path="/addOpportunity/review"
+                title={getTitle("Opportunity Review & Submit")}
+                layout={AuthLayout}
+                component={ReviewOpportunity}
+              />
+              <AppRoute
+                protected
+                path="/addOpportunity/success"
+                title={getTitle("Opportunity Submitted!")}
+                layout={AuthLayout}
+                component={ReviewSubmitted}
+              />
+              <AppRoute path="/investment/*:path" component={OpportunityPage} />
+              <AppRoute
+                protected
+                path="/investment/*:path/approve"
+                layout={AuthLayout}
+                component={OpportunityApprovePage}
+              />
```

Comment:
Yeah, I am just going off what I heard from Dan, as we are rooted at **.bc.ca/investments/

So it will look like `**.bc.ca/investments/investment/asd-asd-1` OR  `**.bc.ca/investments/opportunity/asd-asd-1` Either looks okay to me.

Reply:
I see, that makes sense. Either sounds fine to me as well in that case.